python/tvm/relay/op/op.py

```diff
@@ -364,7 +364,7 @@ def register_shape_func(op_name, data_dependant, shape_func=None, level=10):
     op_name : str
         The name of the op.
 
-    data_dependant : bool
+    data_dependant : bool or list of bool
```

Comment:
Let's have better doc for this so that users know that it could be a list of booleans that indicates if each of the input is data dependent of not.